### ISSUE TITLE
Maintenance addictions dont work.

### ISSUE THREAD

**Issue by remuluson2** [tgstation/tgstation#60737]
Reporting client version: 513.1542

## Round ID:
167317

## Reproduction:
1. Consume about 40+ units of maintenance tar.
2. No addictions at all.

I am not sure if its just meant to take about 400 units to get addicted but i dont think it is.

**Comment by timothymtorres** [comment]
Relevant code for maint drug addiction is here:

https://github.com/tgstation/tgstation/blob/55b92786dcf0f9c5172efd111a6ecbf83ec0dae2/code/modules/reagents/withdrawal/generic_addictions.dm#L94-L161

Relevant code for maint tar is here:

https://github.com/tgstation/tgstation/blob/df5e4aeead63b475934087d10f584771fd62375a/code/modules/reagents/chemistry/reagents/drug_reagents.dm#L413-L436

**Comment by lizardqueenlexi** [comment]
I don't think this is actually a bug - maintenance tar just gives such a low amount of addiction points that it won't addict you unless you take absurd amounts of it. Whether this _should_ be the case is another matter, of course.

**Comment by Sealed101** [comment]
^ they're all pretty low, consuming about 40 units of maintenace tar only gives 200 points when the minimum addiction threshold for maintenance drugs is 600
https://github.com/tgstation/tgstation/blob/8861aef56d487ad7b7512998a5d0d3ece965ba25/code/modules/reagents/chemistry/reagents/drug_reagents.dm#L388-L396
https://github.com/tgstation/tgstation/blob/8861aef56d487ad7b7512998a5d0d3ece965ba25/code/modules/reagents/chemistry/reagents/drug_reagents.dm#L416-L425
https://github.com/tgstation/tgstation/blob/8861aef56d487ad7b7512998a5d0d3ece965ba25/code/modules/reagents/chemistry/reagents/drug_reagents.dm#L446-L453
